dataplane/standalone/lucius/lucius.go

```diff
@@ -33,7 +34,11 @@ import (
 	fwdpb "github.com/openconfig/lemming/proto/forwarding"
 )
 
-var port = flag.Int("port", 50000, "Port for api server")
+var (
+	port          = flag.Int("port", 50000, "Port for api server")
+	configFile    = flag.String("config_file", "", "Path to config file")
+	portMapString = flag.String("port_map", "", "Map of Linux interface to modeled port names")
```

Comment:
.. in comma-sep string format. Or at least show an example.

Reply:
done